tests/vcs/VCSCommitMetadataBearTest.py

```diff
@@ -0,0 +1,112 @@
+import os
+import platform
+import shutil
+import unittest
+import unittest.mock
+from pathlib import Path
+from queue import Queue
+from tempfile import mkdtemp
+
+from bears.vcs.VCSCommitMetadataBear import (VCSCommitMetadataBear,
+                                             CommitResult, COMMIT_TYPE)
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+
+class FakeCommitBear(GitCommitMetadataBear):
+    @classmethod
+    def check_prerequisites(cls):
+        return True
+
+    def get_head_commit(self):
+        return ('This is the fake head commit', '')
+
+
+@generate_skip_decorator(VCSCommitMetadataBear)
+class InvalidCommitResultTest(unittest.TestCase):
```

Comment:
I still don't understand what's this class testing. Could you explain it? if it's not needed remove it

Reply:
Test for line 102-103 of `VCSCommitMetadataBear` . It gave 95 % coverage otherwise.

Reply:
Required thing in this class is the function `def test_get_head_commit_sha()`

Reply:
Ah, okay it makes sense for me now